gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.html

```diff
@@ -45,7 +45,10 @@ <h1>Password Policies</h1>
       <ngx-datatable-column name="Default" [flexGrow]="1" [resizeable]="false" [sortable]="false"></ngx-datatable-column>
       <ngx-datatable-column [flexGrow]="1" [resizeable]="false" [sortable]="false">
         <ng-template let-row="row" ngx-datatable-cell-template>
-          <a [routerLink]="row.id"><mat-icon svgIcon="gio:edit-pencil" matTooltip="Edit policy" class="edit-policy-pencil" /></a>
+          <a [routerLink]="row.id"><mat-icon svgIcon="gio:edit-pencil" matTooltip="Edit policy" class="policy-icon" /></a>
+          <a (click)="$event.preventDefault(); deletePolicy(row.id)" *ngIf="canDelete"
```

Comment:
I would suggest adding prevent default in deletePolicy method

<a (click)="deletePolicy(e, row.id)" *ngIf="canDelete"